cmd/pbstatuser/main.go

```diff
@@ -166,6 +175,8 @@ func checkSourceAvailabilityGCP(ctx context.Context) {
 			sr.Status = kafka.StatusAvaliable
 			chSend <- sr
 		}
+		metrics.IncTotalAvailablilityCheckReqs(models.ProviderTypeGCP, "statuser", sr.Status, err)
+		metrics.ObserveAvailablilityCheckReqsDuration(models.ProviderTypeGCP, start, err)
 	}
 }
```

Comment:
Wait there is a bug, you start the duration at the application launch, not when you receive new variable from the channel.